bot/exts/halloween/spooknamerate.py

```diff
@@ -320,12 +329,25 @@ def in_allowed_month() -> bool:
         """Returns whether running in the limited month."""
         return datetime.utcnow().month == month_to_limit_to
 
+    @staticmethod
+    def ping(id: str) -> str:
```

Comment:
Are you sure you're giving it string of the id?
Worst case; Youcan just type this to be Union[str, int] as it will work with both.